coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
         try:
             if len(result.affected_code) == 0:
+                args['affected_code'] = None
                 print(format_str.format(file=None,
                                         line=None,
                                         end_line=None,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
-                                        **result.__dict__))
+                                        **args))
                 continue
 
             for range in result.affected_code:
+                args['affected_code'] = range
```

Comment:
this looks wrong, if there's a `SourceRange`, why do you reassign it to the `range` built-in function object? It think you should just delete this line^^